clang/include/clang/Driver/Job.h

```diff
@@ -319,6 +319,7 @@ class JobList {
   /// Clear the job list.
   void clear();
 
+  list_type &getJobs() { return Jobs; }
```

Comment:
Could you please enforce that the pre-existing callers use the `const` version? There's at least one `const`-less `auto&` context inside `Driver::BuildJobs()` where it would be preferable to stick to the older behavior; potentially other places as well.

Reply:
fair point, I'll rename the function with a more explicit name rather than using an overload.

Reply:
Renamed this to `getJobsForOverride`